packages/typespec-test/test/multiclient_modular/generated/typespec-ts/package-files/package.json

```diff
@@ -0,0 +1,133 @@
+{
+  "name": "@msinternal/multiclient",
+  "version": "1.0.0-beta.1",
+  "description": "\"Multi Client Service\"",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "react-native": { "./dist/index.js": "./dist-esm/src/index.js" },
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./types/src/index.d.ts",
+      "require": "./dist/index.cjs",
+      "import": "./dist-esm/src/index.js"
+    },
+    "./api": {
+      "types": "./types/src/api/index.d.ts",
+      "import": "./dist-esm/src/api/index.js"
+    },
+    "./rest": {
+      "types": "./types/src/rest/index.d.ts",
```

Comment:
I was wondering if we could export the sub-client directly, so we could avoid to rename anything.

```
 "./rest/bar": {
      "types": "./types/src/rest/bar/index.d.ts",
       "import": "./dist-esm/src/rest/bar/index.js"
 },
 "./rest/foo": {
      "types": "./types/src/rest/foo/index.d.ts",
       "import": "./dist-esm/src/rest/foo/index.js"
 }
```

Reply:
Not a fan of this proposal, even if the package is called multi client, that multiple sub clients are act as a whole package with full functionalities. We use /api or /rest to provide different customer with different user experiences at their needs, I don't think we want to differentiate our customer because of which sub client they are using. @bterlson @mpodwysocki do you have any suggestions on this ?